### PR TITLE
Mybike

### PR DESCRIPTION
**Pull request by Kate-Khon**
Background-color after deploy really spoils the view of "compare bikes" section :(

- [DEMO LINK](https://kate-khon.github.io/layout_miami/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,408 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
```
Comment (illiarerun) [inline src/index.html:6]: Wrong title

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,408 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
```
Comment (illiarerun) [inline src/index.html:11]: Checklist 1st item, add favicon

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,408 @@
...
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="/" class="logo">
```
Comment (illiarerun) [inline src/index.html:18]: Checklist 5th and 6th items
image

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,408 @@
...
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
```
Comment (illiarerun) [inline src/index.html:17]: Icon size should change based on resolution
image

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,408 @@
...
+
+              <p class="about__paragraph">
+                Shift your ride, not gears. Find the fastest way to move in the city as the bike adapts intuitively to power the speed you need.
+              </p>
+
+              <!-- <p class="about__paragraph about__paragraph--important">
+                We will never be late!
+                <br>
+                Guaranteed secure transaction - 100%
+              </p> -->
```
Comment (illiarerun) [inline src/index.html:117]:
```suggestion
```

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,408 @@
...
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section compare-bikes" id="compare-bikes">
+        <div class="container">
+          <h2 class="page__section-title">
+            Compare Bikes
+          </h2>
```
Comment (illiarerun) [inline src/index.html:127]: Price text should be on the left
image

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,408 @@
...
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section details" id="details">
+        <div class="container">
+          <h2 class="page__section-title">
+            The Details
+          </h2>
```
Comment (illiarerun) [inline src/index.html:204]: Checklist 8th item
image

File: src/index.html [hunk 8]
```diff
@@ -1,13 +1,408 @@
...
+                <a
+                  href="https://goo.gl/maps/pin2mxFZiG21x19Q8"
+                  class="contact-us__info-link"
+                >
+                  400 first ave. n.
+                  <br>
+                  suite 700
+                  <br>
+                  Minneapolis, MN 55401
+                </a>
```
Comment (illiarerun) [inline src/index.html:395]: Checklist 9th item

File: src/index.html [hunk 9]
```diff
@@ -1,13 +1,408 @@
...
+
+      <section class="page__section contact-us" id="contact-us">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-5">
+              <form action="#">
```
Comment (illiarerun) [inline src/index.html:326]: Checklist 15th item
image

File: src/index.html [hunk 10]
```diff
@@ -1,13 +1,408 @@
...
+                >
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="
+                    contact-us__field
+                    contact-us__field--last
+                    form-field
+                    form-field--textarea"
+                ></textarea>
```
Comment (illiarerun) [inline src/index.html:347]: Checklsit 14th item

File: src/index.html [hunk 11]
```diff
@@ -1,13 +1,408 @@
...
+            <h1 class="header__title">
+              Take the Streets
+            </h1>
+          </div>
+        </div>
+
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
```
Comment (illiarerun) [inline src/index.html:44]: Checklist 18th item
image

File: src/index.html [hunk 12]
```diff
@@ -1,13 +1,408 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header">
```
Comment (illiarerun) [inline src/index.html:14]: Header looks wrong on mobile
image

**Review by illiarerun — CHANGES_REQUESTED** [review]
Nice start, now let's apply fixes:

**Review by Olexandr6 — APPROVED** [review]
Looks great🔥

Add hover effect:
image

**Pull request closed without merging**